### ISSUE TITLE
Donation rejected

### ISSUE THREAD

**Issue by ariegrossman** [xmrig/xmrig#2920]
2022-02-03 20:27:57.777]  net      dev donate started
[2022-02-03 20:27:57.777]  net      new job from donate.ssl.xmrig.com:443 diff 12456 algo ghostrider height 241895
[2022-02-03 20:27:59.029]  cpu      rejected (275/0) diff 12456 "Invalid params" (171 ms)
[2022-02-03 20:28:00.942]  cpu      rejected (275/0) diff 12456 "Invalid params" (169 ms)
[2022-02-03 20:28:02.597]  cpu      rejected (275/0) diff 12456 "Invalid params" (181 ms)
[2022-02-03 20:28:06.613]  net      new job from donate.ssl.xmrig.com:443 diff 12456 algo ghostrider height 241895
[2022-02-03 20:28:07.863]  cpu      rejected (275/0) diff 12456 "Invalid params" (173 ms)
[2022-02-03 20:28:12.795]  cpu      rejected (275/0) diff 12456 "Invalid params" (172 ms)
[2022-02-03 20:28:17.843]  cpu      rejected (275/0) diff 12456 "Invalid params" (999 ms)
[2022-02-03 20:28:24.424]  cpu      rejected (275/0) diff 12456 "Invalid params" (171 ms)
[2022-02-03 20:28:32.744]  cpu      rejected (275/0) diff 12456 "Invalid params" (175 ms)
[2022-02-03 20:28:35.540]  cpu      rejected (275/0) diff 12456 "Invalid params" (169 ms)
[2022-02-03 20:28:44.651]  cpu      rejected (275/0) diff 12456 "Invalid params" (173 ms)
[2022-02-03 20:28:44.718]  cpu      rejected (275/0) diff 12456 "Invalid params" (169 ms)
[2022-02-03 20:28:49.139]  miner    speed 10s/60s/15m 2765.3 2767.7 3302.3 H/s max 10876.8 H/s avg 3047.1 H/s
[2022-02-03 20:28:57.777]  net      dev donate finished

**Comment by xmrig** [comment]
#2910

**Comment by Lumi412** [comment]
yes i have been getting the same error.  i believe it started version 6.16.2.  i have been getting the error in every version since then, including 6.16.4
